build/make-initrd.sh

```diff
@@ -39,7 +38,10 @@ do
 	ln -sf /sbin/iptables /tmp/hyperstart-rootfs/${bin}
 done
 
-ldd /tmp/hyperstart-rootfs/init | while read line
+LDD_BINARIES=(/init /sbin/ipvsadm /sbin/iptables)
+for bin in ${LDD_BINARIES[@]}
```

Comment:
it is recommended to use the libs provided by the src-tree or built from the src-tree.
currently /init use the host libs, it will be also fixed later.